### PR TITLE
Cleanup of rest of COGNITE_ prefixed env vars

### PR DESCRIPTION
**Pull request by haakonvt**
## Description
Removed the last rest of `COGNITE_` prefixed env.vars.

Also had to make some changes to shut `mypy` up...

### CODE REVIEW

File: cognite/client/data_classes/geospatial.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 from cognite.client import utils
 from cognite.client.data_classes._base import CogniteResource, CogniteResourceList
+from cognite.client.utils._auxiliary import to_snake_case
```
Comment (haakonvt) [inline cognite/client/data_classes/geospatial.py:7]: This entire file was just `mypy` complaining:
```
cognite/client/data_classes/geospatial.py:98: error: Cannot determine type of
"to_snake_case"  [has-type]
        PRE_DEFINED_SNAKE_CASE_NAMES: Set[str] = {utils._auxiliary.to_snak...
```

File: tests/conftest.py [hunk 2]
```diff
@@ -81,9 +80,12 @@ def rsps():
 
 @pytest.fixture
 def disable_gzip():
-    os.environ["COGNITE_DISABLE_GZIP"] = "1"
-    yield
-    del os.environ["COGNITE_DISABLE_GZIP"]
+    try:
```
Comment (erlendvollset) [inline tests/conftest.py:83]: try shouldn't be necessary, I believe. Pytest executes the teardown even when exceptions are raised

**Review by erlendvollset — APPROVED** [review]
Excellent! Thanks for cleaning that up 🐍💥

File: tests/conftest.py (hunk 2 above)
Comment (haakonvt) [reply tests/conftest.py:83]: Oh, thats cool, TIL 🤩

**Pull request merged**